### PR TITLE
specify minimum tls version

### PR DESCRIPTION
**Pull request by Freyert**
I kept receiving 403s from https://fs.blog/feed/ . It turns out that it's probably because cloudflare blocks you unless you specify a minimum TLS version.

TLS 1.3 might be too high, but it solved my problem.

### CODE REVIEW

**Comment by guyfedwards** [comment]
Thanks for the contribution @Freyert. I think we should have the tls version be configurable as 1.2 is still supported. I suggest adding a `minTLSVersion` field to `config.Feed` which can then be used in Fetch.

**Comment by Freyert** [comment]
@guyfedwards agreed! I'll work it in this weekend.

File: internal/config/config.go [hunk 1]
```diff
@@ -138,6 +143,13 @@ func (c *Config) Load() error {
 	c.Feeds = fileConfig.Feeds
 	c.Openers = fileConfig.Openers
 
+	if fileConfig.HTTPOptions != nil {
+		if _, err := TLSVersion(fileConfig.HTTPOptions.MinTLSVersion); err != nil {
+			return err
```
Comment (Freyert) [inline internal/config/config.go:148]: This doesn't seem to work as expected. I get unhelpful output like this if I set it to some unsupported string:

```
flax% go run cmd/nom/main.go
Usage:
  main [OPTIONS]

Application Options:
  -v, --verbose      Show verbose logging
  -n, --number=      Number of results to show
  -p, --pager=       Pager to use for longer output. Set to false for no pager
      --config-path= Location of config.yml
  -f, --feed=        Feed(s) URL(s) for preview
      --version      Display version information

Help Options:
  -h, --help         Show this help message
exit status 1
```

Comment (Freyert) [reply internal/config/config.go:148]: What I would expect is to see a "config error" message and exit status 1.

Comment (guyfedwards) [reply internal/config/config.go:148]: if you run `go run cmd/nom/main.go -v` do you see the error? This is a gotcha with the `--verbose` flag showing errors. Needs to be refactored out at some point

**Comment by guyfedwards** [comment]
@Freyert are you happy with this? Looks good to me

**Comment by guyfedwards** [comment]
ping @Freyert

**Comment by Freyert** [comment]
👍 it's good for me.

**Comment by guyfedwards** [comment]
Thanks @Freyert

**Pull request merged**